dev/com.ibm.ws.webcontainer/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -167,4 +167,7 @@ servletDestroyWaitTime.name=Wait for an active request to complete
 servletDestroyWaitTime.desc=Wait time in seconds for an active request to complete when the owning application is stopped. The default value is 60 seconds. The equivalent custom property in the full application server profile is com.ibm.ws.webcontainer.servletDestroyWaitTime.
 
 servletPathForDefaultMapping.name=Set the servlet path value for a default mapping
-servletPathForDefaultMapping.desc=Set the servlet path value to the request URI minus the context path. The path information is null when a servlet is used as a default mapping. The default value is true for version 4.0 or later of the servlet feature. It is false for other servlet features. When mapping is to the /* pattern, the servlet path is empty and the path information starts with a leading slash (/).
\ No newline at end of file
+servletPathForDefaultMapping.desc=Set the servlet path value to the request URI minus the context path. The path information is null when a servlet is used as a default mapping. The default value is true for version 4.0 or later of the servlet feature. It is false for other servlet features. When mapping is to the /* pattern, the servlet path is empty and the path information starts with a leading slash (/).
+
+throwExceptionWhenUnableToCompleteOrDispatch.name=Throw an Illegal State Exception when unable to complete or dispatch async request
+throwExceptionWhenUnableToCompleteOrDispatch.desc=Throw an Illegal State Exception when an async request can not be completed or dispatched. The default is true. If it is necessary and verified that the async complete or dispatch method must return even if the call does not succeed, consider set the property to false.
```

Comment:
Change "Throw an Illegal State Exception when an async request can not be completed or dispatched. The default is true. If it is necessary and verified that the async complete or dispatch method must return even if the call does not succeed, consider set the property to false." to "Create an illegal state exception when an asynchronous request cannot be completed or dispatched. The default is true. If the asynchronous request must complete or the dispatch method must return even if the call does not succeed, set the property to false.".